.gitignore

```diff
@@ -1,4 +1,5 @@
 __pycache__
+ext/dcos_installer/packages/
```

Comment:
why are we adding this?

At the very least seems unrelated. None of the stuff any of our tooling does ends up here.